shared/dataRepository.ts

```diff
@@ -48,8 +50,11 @@ const generateFilePath = (time: NzTimeIso) => {
   return `${CONSTANTS.dataFolder}/${date}.txt`;
 };
 
-const getLatestVaccineDataFilePath = async () => {
+const getVaccineDataFilePath = async (period: "latest" | "yesterday") => {
```

Comment:
Should this use the type??

```suggestion
const getVaccineDataFilePath = async (period: DataPeriod) => {
```

Reply:
Yep my bad, have fixed this